python/ql/src/Security/CWE-295/RequestWithoutValidation.qhelp

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>
+Encryption is key to the security of most, if not all, online communication.
+Using TLS can enusre that neither party in the communication is an interloper.
+For this reason, is is unwise to disable the verification that TLS provides.
+<code>requests</code> provides verification by default, and it is only when
+explicitly turned off using <code>verify=False</code> that no verification occurs.
+</p>
+</overview>
+
+<recommendation>
+<p>
+Never use <code>verify=False</code> when making a request.
+</p>
+</recommendation>
+
+<example>
+<p>
+The example shows two unsafe calls to <a href="https://semmle.com">semmle.com</a>, followed by various safe alternatives.
```

Comment:
Does this text also cover the wrapper?

Reply:
Yes. The code should be self explanatory.